### PR TITLE
Added technical documentation

### PR DESCRIPTION
**Pull request by thanrobinson**

### CODE REVIEW

**Comment by thanrobinson** [comment]
[HII Power Read Me](https://trello.com/c/XaWlJwKt/347-hii-power-read-me)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
-# task_hii_power
-Earth Engine task for calculating power driver
+HII POWER DRIVER
+---------------
+
+## What does this task do?
+
+This task calculates the (unitless) "influence" of power on the terrestrial surface as one of the key drivers for a combined [Human Influence Index](https://github.com/SpeciesConservationLandscapes/task_hii_weightedsum). "Influence" is a pressure score based on the intensity of electricity usage as measured by "Night Time Lights" datasets. An inter-calibrated nighttime lights dataset with values ranging from 0 - 63 was transformed to a pressure score of 0 - 10 using equal interval quantiles, calculated form the earliest available image (1992). These were:
```
Comment (saanobhaai) [inline README.md]: - calculated from
- is transformed -- should do a global search for remaining past/passive verbs

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+     "4": {"value": 4, "min": 7, "max": 7},
+     "5": {"value": 5, "min": 8, "max": 9},
+     "6": {"value": 6, "min": 10, "max": 11},
+     "7": {"value": 7, "min": 12, "max": 16},
+     "8": {"value": 8, "min": 17, "max": 30},
+     "9": {"value": 9, "min": 31, "max": 62},
+     "10": {"value": 10, "min": 63, "max": 63},
+ ```
+
+### Input Dataset Callibration
```
Comment (saanobhaai) [inline README.md]: Calibration (single l; do a global search/replace)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,56 @@
...
+Two distinct nightlight datasets are integrated to produce an annually calibrated dataset of nightlights from 1992 through the present. These the [Defense Meteorological Satellite Program (DMSP)/Operational Linescan System (OLS)](https://eogdata.mines.edu/products/dmsp/) and the [Visible Infrared Imaging Radiometry Suite (VIIRS) on the Suomi National Polar-orbiting Partnership Satellite](https://eogdata.mines.edu/products/vnl/). DMSP provides data from 1992 - 2013 and VIIRS provide data from 2012 through the present.
```
Comment (saanobhaai) [inline README.md]: The HII power driver (0-10) is calculated from the previous calendar year's calibrated nightlights dataset, which is produced on demand by the task. Two distinct source nightlight datasets are used to calculate this calibrated version following the methods below; these are the [Defense Meteorological Satellite Program (DMSP)/Operational Linescan System (OLS)](https://eogdata.mines.edu/products/dmsp/) and the [Visible Infrared Imaging Radiometry Suite (VIIRS) on the Suomi National Polar-orbiting Partnership Satellite](https://eogdata.mines.edu/products/vnl/). DMSP provides data from 1992 - 2013 and VIIRS provides data from 2012 through the present.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,56 @@
...
+
+Inconsistencies within the DMSP time series requires implementing intra-calibration within the DMPS dataset. Here the intra-calibrated dataset produced by [Li et al. 2020](https://www.nature.com/articles/s41597-020-0510-y) is used. Key differences between the DMSP and VIIRS datasets also require inter-calibration of the two datasets. In addition to the calibration process to match VIIRS to DMSP, VIIRS requires substantial noise reduction. These key differences are:
```
Comment (saanobhaai) [inline README.md]: require

File: README.md [hunk 5]
```diff
@@ -1,2 +1,56 @@
...
+
+&emsp; &emsp; &emsp; t = (|lat|<sup>4</sup> / 60<sup>4</sup>) x (t<sub>max</sub> - t<sub>min</sub>) + (t<sub>min</sub>)
+
+&emsp; &emsp; &emsp; &emsp; Where: <br />
+&emsp; &emsp; &emsp; &emsp; t is the final threshold image <br />
+&emsp; &emsp; &emsp; &emsp; lat is a latitude image <br />
+&emsp; &emsp; &emsp; &emsp; t<sub>min</sub> is the minimum threshold <br />
+&emsp; &emsp; &emsp; &emsp; t<sub>max</sub> is the maximum threshold
+
+3. Regression coeffecients were calculated to convert VIIRS values to DMPS values by comparing pixel values of areas with stable lights through time. These were determined by selecting pixels from the DMSP time series from 2000 - 2012 that had a standard deviation less than 2. A stratified sample of 200 pixel values for each available DMSP value was calculated. The resulting equation to transform VIIRS values to match DMSP was determined to be:
```
Comment (saanobhaai) [inline README.md]: DMSP

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Just some typos and suggested rewording

**Pull request merged**